DQM/L1TMonitor/python/L1TdeStage2uGMT_cff.py

```diff
@@ -83,6 +83,7 @@
     verbose = cms.untracked.bool(False),
     enable2DComp = cms.untracked.bool(True), # When true eta-phi comparison plots are also produced
     displacedQuantities = cms.untracked.bool(False),
+    ignoreBin = cms.untracked.vint32(ignoreBins),
```

Comment:
Please check, but I think that the default was with no bins to ignore (and it gets modified below in the different specific implementations):
```suggestion
    ignoreBin = cms.untracked.vint32(),
```